[CI] MedianAbsoluteDeviationLongAggregatorFunctionTests testSimpleSmallInput failing

**Build Scans:**
- [elasticsearch-intake #30319 / part3](https://gradle-enterprise.elastic.co/s/txwgzsq5l6ku4)
- [elasticsearch-periodic #11329 / concurrent-search-tests](https://gradle-enterprise.elastic.co/s/nlpjg45oct6ty)
- [elasticsearch-pull-request #99978 / part-3](https://gradle-enterprise.elastic.co/s/yqxi23l5cuwzc)
- [elasticsearch-periodic #11329 / openjdk21_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/vj3kky5loieam)
- [elasticsearch-intake #30307 / part3](https://gradle-enterprise.elastic.co/s/riwb2pulgh4ya)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:compute:test" --tests "org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationLongAggregatorFunctionTests.testSimpleSmallInput" -Dtests.seed=CAF3595A0431A881 -Dtests.locale=et-EE -Dtests.timezone=Asia/Muscat -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationLongAggregatorFunctionTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testSimpleSmallInput'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <23.0>
     but: was <15.5>
```

**Issue Reasons:**
- [main] 5 failures in test testSimpleSmallInput (2.1% fail rate in 242 executions)
- [main] 2 failures in step part3 (5.6% fail rate in 36 executions)
- [main] 2 failures in pipeline elasticsearch-intake (5.6% fail rate in 36 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -540,18 +540,6 @@ tests:
 - class: org.elasticsearch.xpack.search.CrossClusterAsyncSearchIT
   method: testRemoteClusterOnlyCCSWithFailuresOnAllShards
   issue: https://github.com/elastic/elasticsearch/issues/136894
-- class: org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationFloatAggregatorFunctionTests
-  method: testSimpleSmallInput
-  issue: https://github.com/elastic/elasticsearch/issues/136902
-- class: org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationLongAggregatorFunctionTests
-  method: testSimpleSmallInput
-  issue: https://github.com/elastic/elasticsearch/issues/136903
-- class: org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationDoubleAggregatorFunctionTests
-  method: testSimpleSmallInput
-  issue: https://github.com/elastic/elasticsearch/issues/136904
-- class: org.elasticsearch.compute.aggregation.MedianAbsoluteDeviationIntAggregatorFunctionTests
-  method: testSimpleSmallInput
-  issue: https://github.com/elastic/elasticsearch/issues/136905
 - class: org.elasticsearch.xpack.esql.qa.mixed.FieldExtractorIT
   method: testTextFieldWithIpSubfieldMalformed {STORED}
   issue: https://github.com/elastic/elasticsearch/issues/136917
@@ -561,9 +549,6 @@ tests:
 - class: org.elasticsearch.xpack.test.rest.XPackRestIT
   method: test {p0=ml/sparse_vector_search/Test sparse_vector search with query vector and pruning config}
   issue: https://github.com/elastic/elasticsearch/issues/136061
-- class: org.elasticsearch.compute.operator.HashAggregationOperatorTests
-  method: testSimpleSmallInput
-  issue: https://github.com/elastic/elasticsearch/issues/136945
 - class: org.elasticsearch.xpack.esql.optimizer.rules.logical.HoistRemoteEnrichTopNTests
   method: testTopNSortExpressionWithinRemoteEnrichAliasing
   issue: https://github.com/elastic/elasticsearch/issues/136957

--- a/x-pack/plugin/esql/compute/test/src/main/java/org/elasticsearch/compute/test/OperatorTestCase.java
+++ b/x-pack/plugin/esql/compute/test/src/main/java/org/elasticsearch/compute/test/OperatorTestCase.java
@@ -73,7 +73,7 @@ public abstract class OperatorTestCase extends AnyOperatorTestCase {
     }
 
     protected int smallInputSize() {
-        return randomIntBetween(1, 100);
+        return randomIntBetween(10, 100);
     }
 
     /**

